infra/basemaps.ts

```diff
@@ -0,0 +1,21 @@
+import { handler as h } from '@basemaps/lambda-tiler';
+import { ConfigProviderMemory, Config, ConfigBundled } from '@basemaps/config';
+import { fsa } from '@chunkd/fs';
+
+process.env['BASEMAPS_ASSEST_LOCATION'] = process.env['ASSETS_LOCATION'];
+
+(async (): Promise<void> => {
```

Comment:
you will need to wait for this inside of `handler` so assign this to a local promise.

Reply:
Can we change the handler to async here?

Reply:
`configLoader.then(() => h(...))`